images/releng/k8s-ci-builder/variants.yaml

```diff
@@ -1,29 +1,29 @@
 variants:
   default:
     CONFIG: default
-    GO_VERSION: '1.21.2'
+    GO_VERSION: '1.21.3'
     OS_CODENAME: 'bullseye'
   next:
     CONFIG: next
-    GO_VERSION: '1.21.2'
+    GO_VERSION: '1.21.3'
     OS_CODENAME: 'bookworm'
   '1.29':
     CONFIG: '1.29'
-    GO_VERSION: '1.21.2'
+    GO_VERSION: '1.21.3'
     OS_CODENAME: 'bullseye'
   '1.28':
     CONFIG: '1.28'
-    GO_VERSION: '1.20.9'
+    GO_VERSION: '1.20.10'
     OS_CODENAME: 'bullseye'
   '1.27':
     CONFIG: '1.27'
-    GO_VERSION: '1.20.9'
+    GO_VERSION: '1.20.10'
```

Comment:
Still see the changes for 1.20.10

Reply:
@cpanato Looks like there was some issue with GitHub, PTAL again